pkg/scheduler/schedule_one.go

```diff
@@ -150,8 +151,9 @@ func (sched *Scheduler) schedulingCycle(ctx context.Context, state *framework.Cy
 			nominatingInfo = clearNominatedNode
 			klog.ErrorS(err, "Error selecting node for pod", "pod", klog.KObj(pod))
 			metrics.PodScheduleError(fwk.ProfileName(), metrics.SinceInSeconds(start))
+			reason = SchedulerError
```

Comment:
we should probably move it to the v1 package.

But I would do it in a separate PR so that we can cherry-pick this as is.

Reply:
Yes, I was plan to.